app/src/main/java/co/kr/soptandroidseminar/view/onboarding/OnBoardingThreeFragment.kt

```diff
@@ -0,0 +1,37 @@
+package co.kr.soptandroidseminar.view.onboarding
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import co.kr.soptandroidseminar.R
+import co.kr.soptandroidseminar.databinding.FragmentOnBoardingThreeBinding
+import co.kr.soptandroidseminar.view.signin.SignInActivity
+
+class OnBoardingThreeFragment : Fragment() {
+    private var _binding: FragmentOnBoardingThreeBinding? = null
+    private val binding get() = _binding!!
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentOnBoardingThreeBinding.inflate(layoutInflater, container, false)
+        skipOnBoarding()
+        return binding.root
+    }
+
+    override fun onDestroyView() {
+        super.onDestroyView()
+        _binding = null
+    }
```

Comment:
```suggestion
    override fun onDestroyView() {
         _binding = null
        super.onDestroyView()
    }
```

super.onDestroyView가 부모 클래스의 super.onDestoryView를 전부 해주는거니 이건 제일 마지막에 호출되는게 좋을 듯

Reply:
호오오오우 반영했읍니다